src/openforms/submissions/tests/form_logic/test_fetching_form_varaiable_values_from_services.py

```diff
@@ -349,6 +349,60 @@ def test_it_sends_the_body_as_json(self, m):
         self.assertIn(("Content-Type", "application/json"), request.headers.items())
         self.assertEqual(request.body, b'"Armour"')
 
+    @requests_mock.Mocker()
+    def test_it_sends_the_body_as_json_with_variables(self, m):
+        m.get("https://httpbin.org/anything", json={"foo": "bar"})
+
+        var = FormVariableFactory.build(
+            service_fetch_configuration=ServiceFetchConfigurationFactory.build(
+                service=self.service,
+                path="anything",
+                body={"foo": "{{some_variable}}"},
+            )
+        )
+
+        _ = perform_service_fetch(var, {"some_variable": "bar"})
+        request = m.last_request
+
+        self.assertIn(("Content-Type", "application/json"), request.headers.items())
+        self.assertEqual(request.body, b'{"foo": "bar"}')
+
+    @requests_mock.Mocker()
+    def test_it_sends_the_body_as_json_with_variables_nested(self, m):
+        m.get("https://httpbin.org/anything", json={"foo": {"nested": "bar"}})
+
+        var = FormVariableFactory.build(
+            service_fetch_configuration=ServiceFetchConfigurationFactory.build(
+                service=self.service,
+                path="anything",
+                body={"foo": {"nested": "{{some_variable}}"}},
+            )
+        )
+
+        _ = perform_service_fetch(var, {"some_variable": "bar"})
+        request = m.last_request
+
+        self.assertIn(("Content-Type", "application/json"), request.headers.items())
+        self.assertEqual(request.body, b'{"foo": {"nested": "bar"}}')
+
+    @requests_mock.Mocker()
+    def test_it_sends_the_body_as_json_with_variables_escape_quotes(self, m):
+        m.get("https://httpbin.org/anything", json={"foo": "&quot;bar"})
+
+        var = FormVariableFactory.build(
+            service_fetch_configuration=ServiceFetchConfigurationFactory.build(
+                service=self.service,
+                path="anything",
+                body={"foo": "{{some_variable}}"},
+            )
+        )
+
+        _ = perform_service_fetch(var, {"some_variable": '"bar'})
+        request = m.last_request
+
+        self.assertIn(("Content-Type", "application/json"), request.headers.items())
+        self.assertEqual(request.body, b'{"foo": "&quot;bar"}')
```

Comment:
```suggestion
        self.assertEqual(request.body, rb'{"foo": "\"bar"}')
```
Maybe check the other values that need to be escaped too:
https://www.json.org/img/string.png